apps/i18n/common/en_us.json

```diff
@@ -1586,6 +1587,7 @@
   "notSignedInBody": "You don't need an account to work on this lesson, but if you want to save your work, remember to sign in or create an account before you get started.",
   "notStarted": "Not started",
   "nPoints": "{numPoints, plural, one {1 point} other {# points}}",
+  "numAttempts": "{numAttempts, plural, one {1 attempt} other {# attempts}}",
```

Comment:
cool! I'd heard this kind of translation was a possibility but never seen it in action before